paramak/__init__.py

```diff
@@ -1,3 +1,16 @@
+try:
+    from importlib.metadata import version, PackageNotFoundError
+except (ModuleNotFoundError, ImportError):
+    from importlib_metadata import version, PackageNotFoundError
```

Comment:
```suggestion
try:
    from importlib.metadata import version, PackageNotFoundError
except (ModuleNotFoundError, ImportError):
    from importlib_metadata import version, PackageNotFoundError
```

The code run in both cases appears to be the same, perhaps we don't need the try: except: here

Reply:
Ah ignore this, I didn't notice the . is replaced by an _ in the second case

Reply:
Ah, this is a bit subtle -- `importlib.metadata` was introduced in the standard library in 3.8, so if you want to support python <=3.7, you then need the third-party backport `importlib_metadata` instead.

Reply:
Going from Python 3.7 to 3.8, importlib_metadata was folded into the importlib package as importlib.metadata, so the purpose of the try/except is to maintain backwards compatibility with Python 3.7 and earlier. As the project is quite dependent on cadquery, and I haven't had much luck getting it to work with anything besides Python 3.8, I'm not sure if backwards compatibility is actually an option.